### PR TITLE
feat(Flow): add Slider parameter renderer

### PR DESCRIPTION
**Pull request by plagoa**
- Added the Slider renderer. 
- Updated the docs.

### CODE REVIEW

File: packages/lab/src/Flow/types/flow.ts [hunk 1]
```diff
@@ -95,7 +98,18 @@ export interface HvFlowNodeSelectParam extends HvFlowNodeSharedParam {
...
 }
 
-export type HvFlowNodeParam = HvFlowNodeSelectParam | HvFlowNodeTextParam;
+export interface HvFlowNodeSliderParam extends HvFlowNodeSharedParam {
+  type: "slider";
+  minPointValue?: HvSliderProps["minPointValue"];
+  maxPointValue?: HvSliderProps["maxPointValue"];
+  markStep?: HvSliderProps["markStep"];
+  divisionQuantity?: HvSliderProps["divisionQuantity"];
+}
```
Comment (zettca) [inline packages/lab/src/Flow/types/flow.ts:107]: Why stop at these 4 props? What if the user needs (for instance, `formatMark`)? We're already forwarding any props to the `HvSlider`, I think we should allow in on the types too
(we might need to Omit some props)

```suggestion
export interface HvFlowNodeSliderParam extends HvFlowNodeSharedParam,
  Omit<HvSliderProps, keyof HvFlowNodeSharedParam>{
  type: "slider";
}
```

File: packages/lab/src/Flow/Node/Parameters/Slider.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+};
+
+const Slider = ({ nodeId, param, data }: SliderProps) => {
+  const { id } = param;
+
+  const reactFlowInstance = useReactFlow();
+
+  const [value, setValue] = useState(data[id]);
+
+  const onSliderchange: HvSliderProps["onChange"] = (val) => {
```
Comment (zettca) [inline packages/lab/src/Flow/Node/Parameters/Slider.tsx:34]: We shouldn't need to control the value of the element internally, we could use `defaultValues={data[id]}`

Also, typo on the capitalisation of `onSliderChange`

```suggestion
  const onSliderChange: HvSliderProps["onChange"] = (val) => {
```

File: packages/lab/src/Flow/stories/Usage.stories.mdx [hunk 3]
```diff
@@ -243,7 +243,18 @@ interface HvFlowNodeSelectParam {
...
 }
 
-type HvFlowNodeParam = HvFlowNodeSelectParam | HvFlowNodeTextParam;
+export interface HvFlowNodeSliderParam {
+  type: "slider";
+  minPointValue: number;
+  maxPointValue: number;
+  markStep: number;
+  divisionQuantity: number;
+}
```
Comment (zettca) [inline packages/lab/src/Flow/stories/Usage.stories.mdx:252]: We can simplify this is we point to the Slider props. Also, the `id` and `label` keys are present in the other params but not this one

Ideally, we wouldn't have to duplicate the types

**Review by francisco-guilherme — APPROVED** [review]

**Comment by plagoa** [comment]
@zettca updated PR.

**Review by zettca — APPROVED** [review]

**Pull request merged**